packaging/debian-sid/not-installed

```diff
@@ -1,4 +1,5 @@
 usr/bin/snap-repair
 usr/bin/snap-failure
 usr/lib/snapd/system-shutdown
-usr/bin/snap-bootstrap
\ No newline at end of file
+usr/bin/snap-bootstrap
+usr/bin/snap-recovery-chooser
```

Comment:
Do you need to handle this in `packaging/snapd.mk`, namely to remove it like so:
```
ifeq ($(with_core_bits),0)
# Remove core 20 recovery chooser
install::
    rm -f $(addprefix $(DESTDIR)$(bindir)/snap-recovery-chooser
endif
```

Reply:
We just don't build it there.

Reply:
Ah, indeed. I was under the impression we build all go binaries. This is fine though I would prefer to have it for completeness, with the hope that this makefile can be used in all packaging at some point.